src/harvest.cpp

```diff
@@ -119,13 +122,30 @@ void harvest_list::check_consistency()
 {
     for( const auto &pr : harvest_all ) {
         const auto &hl = pr.second;
+        const std::string hl_id = hl.id().c_str();
+        auto error_func = [&]( const harvest_entry & entry ) {
+            std::string errorlist = "";
+            if( !( item::type_is_defined( entry.drop ) || ( entry.type == "bionic_group" &&
+                    item_group::group_is_defined( entry.drop ) ) ) ) {
+                errorlist += entry.drop;
```

Comment:
This is string concatenation, you'll end up with a single string that contains "foonull type" ("foo" from `entry.drop`, 
 "null type" from the string literal below) - not space, no other thing to distinguish parts of it.

I suggest you add a few spaces to separate the values stored in `errorlist`.